packages/dds/tree/src/feature-libraries/flex-tree/flexTreeTypes.ts

```diff
@@ -265,52 +275,6 @@ export interface FlexTreeField extends FlexTreeEntity<FlexFieldSchema> {
 export interface FlexTreeMapNode<in out TSchema extends FlexMapNodeSchema>
```

Comment:
does map node still make sense as a name if the only thing this interface adds is a schema?

Reply:
It doesn't actually add schema as a member, it provides a more specific type for the existing schema member. Removing this causes unions of node types to narrow with .is incorrectly since if all nodes have the same API, node.is(schema) returning tells tells typescript the value is not a node at all, which is wrong and broke some tests.